srcpkgs/php-zip/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'php-zip'
+pkgname=php-zip
+version=1.15.4
+revision=1
+wrksrc="zip-${version}"
+build_style=gnu-configure
+make_check_target=test
+hostmakedepends="autoconf pcre2-devel php-devel"
+makedepends="libzip-devel php-devel"
+depends="libzip php"
```

Comment:
libzip is detected via shlibs, it does not need to be explicitely defined as a dependency.

Reply:
Would that be the same for the makedepends line or do I need to explicitly define it there?

Reply:
It needs to be explicitely listed in makedepends as it can not be automatically detected there. After compiling the binaries, xbps-src will check which shared libraries the produced binaries link against and add these dependencies automatically.